inception-workload/src/main/java/de/tudarmstadt/ukp/inception/workload/model/Workload_Manager.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.workload.model;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+
+/**
+ * A persistence object for the workflow and workload properties of each project.
+ */
+@Entity
+@Table(name = "workload_manager")
+public class Workload_Manager implements Serializable
+{
+    private static final long serialVersionUID = -3289504168531309833L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "project", nullable = false)
+    private Project project;
+
+    @Column(columnDefinition = "VARCHAR(50)")
```

Comment:
Why only 50 characters?

Reply:
Should be enough for anything to come. Of cource, i can put a VARCHAR(255) however I do not think this might be necessary.

Reply:
Normally, we use VARCHAR(255). I'd tend to stick with that although we have a bit of trouble with it with MySQL:

> The difference between utf8 and utf8mb4 charset is the bytes requires to store each characters. utf8 requires 3 bytes, while utf8mb4 requires 4 bytes. This means using utf8mb4 charset in a table with innodb engine with innodb_large_prefix disabled, at most 191 characters in a string column must be used.